### ISSUE TITLE
Cannot load images with the latest submission of the master branch

### ISSUE THREAD

**Issue by xuzhijvn** [razonyang/hugo-theme-bootstrap#199]
![image](https://user-images.githubusercontent.com/20964259/131452059-76ed8597-2ad9-45ec-94fa-0752786656f6.png)

**Comment by razonyang** [comment]
Hi, could you please post the information of git log?

```
git log -5 --pretty=oneline
```

I' sure that it's a bug introduced by 2e62fd1e17bdc391ba5eece2ff95e4619676eac3, and was fixed by 02f941a5c7e0dfaead18215d67bcfe86eab08f7c.

**Comment by xuzhijvn** [comment]
> Hi, could you please post the information of git log?
> 
> ```
> cd themes/hugo-theme-bootstrap
> git log -5 --pretty=oneline
> ```
> 
> I ~am sure~ think that it's a bug introduced by [2e62fd1](https://github.com/razonyang/hugo-theme-bootstrap/commit/2e62fd1e17bdc391ba5eece2ff95e4619676eac3), and was fixed by [02f941a](https://github.com/razonyang/hugo-theme-bootstrap/commit/02f941a5c7e0dfaead18215d67bcfe86eab08f7c).


```
D:\WorkSpace\Webstorm\xuzhijvn.github.io\themes\hugo-theme-bootstrap>git log -5 --pretty=oneline
3b16d098e8cfe0201027e39e01f4bc7c3ded3841 (HEAD -> master, origin/master, origin/HEAD) Update _post.scss
02f941a5c7e0dfaead18215d67bcfe86eab08f7c Minor changes
b8b4d7c8f057d77f33f0c428269a0a9286d637c8 Update the example site
2e62fd1e17bdc391ba5eece2ff95e4619676eac3 Add support for page image resources
4239fbd44a9b034ccdab44f4d0510a31d719eccf Fix #197
```

**Comment by razonyang** [comment]
Fixed by e620bca, sorry about that.

**Comment by xuzhijvn** [comment]
> Fixed by [e620bca](https://github.com/razonyang/hugo-theme-bootstrap/commit/e620bca56f89194303c83e9a6811e245e3650c61), sorry about that.

thanks, your are the best, man
